trillian/migrillian/main.go

```diff
@@ -88,11 +88,13 @@ func main() {
 
 	ctx := context.Background()
 	cctx, cancel := context.WithTimeout(ctx, *dialTimeout)
-	conn, err := grpc.DialContext(cctx, cfg.TrillianUri,
+	// TODO(pavelkalinnikov): Replace LogBackendName by the BackendSpec extracted
+	// from MigrillianConfig.
+	conn, err := grpc.DialContext(cctx, cfg.LogBackendName,
```

Comment:
You might have to revisit blocking here. CTFE blocks if it has a single backend as it can't do any work until connected. However multiple backends means partial availability is possible. Can be addressed later but it's worth thinking about.

Reply:
Sure, I have an upcoming PR on top of this one which addresses this point.